support-frontend/assets/helpers/forms/paymentIntegrations/readerRevenueApis.ts

```diff
@@ -135,11 +135,11 @@ export type RegularPaymentFields =
 	| RegularAmazonPayPaymentFields;
 export type RegularPaymentRequestAddress = {
 	country: IsoCountry;
-	state: UsState | CaState | null;
-	lineOne: Option<string>;
-	lineTwo: Option<string>;
-	postCode: Option<string>;
-	city: Option<string>;
+	state?: UsState | CaState | null;
+	lineOne?: Option<string>;
+	lineTwo?: Option<string>;
+	postCode?: Option<string>;
+	city?: Option<string>;
```

Comment:
[`?:` is a more idiomatic TypeScript way of having optional properties](https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties).

I'll put in another PR to remove the `Option` as it's a little more far reaching.